docs/architecture/modernize-desktop/deploy-modern-applications.md

```diff
@@ -0,0 +1,220 @@
+
+![Configuring Packages](media\deploying-modern-desktop-applications\configuring-packages.png)
+
+There you can select the different architectures you want to target as you can include as many as you want into de the same MSIX package.
+
+The final step is to declare where you want to deploy the final installation assets.
+
+![Configure Update Settings](media\deploying-modern-desktop-applications\configure-update-settings.png)
+
+You can choose to use a web server of a shared UNC path on your enterprise file servers. Pay attention to the settings you can specify to set up how you want to update your application. We will be discussing about application updates in the next section.
+
+For a detail step-by-step guide refer to: [Package a desktop app from source code using Visual Studio](https://docs.microsoft.com/windows/msix/desktop/desktop-to-uwp-packaging-dot-net)
+
+## Auto Updates in MSIX
+
+The Windows Store has a great updating mechanism using Windows Update. In most enterprise scenarios, you don't use the Store to distribute your desktop apps so you need a similar way to configure updates for your application and pull them to your users.
+
+Using a combination of Windows 10 features and MSIX packages, you can provide a great updating experience for your users. In fact, the user does not need to be technical at all but still benefit from a seamless application update experience.
+
+You can configure your updates to interact with the user in two different ways:
+
+- User prompted updates: The OS shows some autogenerated nice UI to notify the user about the application is about to install. It builds this UI based on the properties you specify on your installation files.
+
+- Silent updates in the background. With this option, your users don't need to be aware of the updates.
+
+You can also configure when you want to perform updates, when the application launches or on a regularly basis. Thanks to the side-loading features, you can even get these updates while the application is running.
+
+When you use this type of deployment, a special file is created called *.appinstaller*. This simple file contains the following sections:
+
+- The location of the *.appinstaller* file
+- The application's main MSIX package properties
+- The update behavior
+
+![.appinstaller file](media\deploying-modern-desktop-applications\appinstaller-file.png)
+
+In combination with this file, Microsoft has designed a special URL protocol to launch the installation process from a link:
+
+```xml
+<a href="ms-appinstaller:?source=http://mywebservice.azureedge.net/MyApp.msix">Install app package </a>
+```
+
+This protocol work on all browsers and launches the installation process with a great user experience on Windows 10. Since the OS manages the installation process, it is aware of the location this application was installed from and tracks all the files affected by the process.
```

Comment:
```suggestion
This protocol work on all browsers and launches the installation process with a great user experience on Windows 10. Since the OS manages the installation process, it's aware of the location this application was installed from and tracks all the files affected by the process.
```